flair/datasets/sequence_labeling.py

```diff
@@ -20,12 +20,10 @@ def __init__(
             test_file=None,
             dev_file=None,
             tag_to_bioes=None,
-            column_delimiter: str = "\s+",
```

Comment:
This seems to revert the addition of column_delimiters and the option to skip the first line. Is this intended?

Reply:
Ah no absolutely not, thanks for catching this.